yourgov/components/FormMobileFoodVendorPermit/task1/FormTask1Step2.tsx

```diff
@@ -38,95 +30,70 @@ export function FormTask1Step2() {
-		resolver: yupResolver(task1Step2FormSchema),
+		resolver: isSavingBeforeExiting
+			? undefined
+			: zodResolver(task1Step2FormSchema),
+		mode: 'onSubmit',
+		reValidateMode: 'onBlur',
 	});
 
 	const onSubmit: SubmitHandler<Task1Step2FormSchema> = async (data) => {
-		setFocusedError(false);
+		if (isSavingBeforeExiting) {
+			return;
+		}
 		await submitStep();
 		setFormState({
 			...formState,
-			task1: { ...formState.task1, step2: { ...data, completed: true } },
+			task1: {
+				...formState.task1,
+				step2: {
+					...data,
+					completed: !isSavingBeforeExiting,
+					started: true,
+				},
+			},
 		});
 	};
 
-	const onError = () => {
-		setFocusedError(false);
-	};
+	const showErrorAlert = hasMultipleErrors(errors);
 
-	// Only show the page alert if there is more than 1 error
-	const hasErrors = Object.keys(errors).length > 1;
-
-	useEffect(() => {
-		if (hasErrors && !focusedError) {
-			errorRef.current?.focus();
-			setFocusedError(true);
-		}
-	}, [hasErrors, focusedError, errors]);
-
-	const showConditionalField = watch('businessStructure') === 'Business';
+	const showAbn = watch('businessStructure') === 'Business';
+	const showAcn = watch('businessStructure') === 'Company';
 
 	useEffect(() => {
 		if (isSubmitted) trigger();
-	}, [trigger, isSubmitted, showConditionalField]);
+	}, [isSubmitted, trigger]);
 
 	return (
 		<FormTask1Container
 			formTitle="Business details"
 			formIntroduction="Your business details must match your business registration."
-			formCallToAction={<FormRequiredFieldsMessage />}
 		>
-			<Stack
-				as="form"
-				gap={3}
-				onSubmit={handleSubmit(onSubmit, onError)}
-				noValidate
-			>
+			<Stack as="form" gap={3} onSubmit={handleSubmit(onSubmit)} noValidate>
 				<FormStack>
-					{hasErrors && (
-						<PageAlert
-							ref={errorRef}
-							tone="error"
-							title="There is a problem"
-							tabIndex={-1}
-						>
-							<Text as="p">
-								Please correct the following fields and try again
-							</Text>
-							<UnorderedList>
-								{Object.entries(errors).map(([key, value]) => (
-									<ListItem key={key}>
-										<TextLink href={`#${key}`} onClick={scrollToField}>
-											{value.message}
-										</TextLink>
-									</ListItem>
-								))}
-							</UnorderedList>
-						</PageAlert>
-					)}
+					{showErrorAlert && <FormPageAlert errors={errors} />}
 					<TextInput
 						id="businessName"
 						label="Business or company name"
-						hint="Hint text"
 						{...register('businessName')}
 						invalid={Boolean(errors.businessName?.message)}
 						message={errors.businessName?.message}
 						required
+						maxWidth="xl"
 					/>
 
 					<TextInput
 						id="tradingName"
 						label="Trading name"
-						hint="Hint text"
 						{...register('tradingName')}
 						invalid={Boolean(errors.tradingName?.message)}
 						message={errors.tradingName?.message}
+						maxWidth="xl"
 					/>
 
 					<ControlGroup
```

Comment:
When this field is in error and you attempt to navigate to it from the page alert, no focus indicator is shown.

Reply:
It's optonal. How do you even get an error in this?

Reply:
Don't fill anything out and press "Save and continue".

Reply:
Oh, the comment is on the radios not the `maxWidth` on the tradingName!

Reply:
Fix for this in this PR https://github.com/agriculturegovau/agds-next/pull/1743